### PR TITLE
Raise error for empty title when adding record (KC-211)

### PR DESCRIPTION
**Pull request by kihneman**

### CODE REVIEW

File: keepercommander/commands/record.py [hunk 1]
```diff
@@ -209,6 +209,8 @@ def execute(self, params, **kwargs):
         if not force:
             if not title:
                 title = input('...' + 'Title: '.rjust(16))
+                if len(title) == 0:
```
Comment (sk-keeper) [inline keepercommander/commands/record.py]: I don't think this change is enough.
The empty title can be created when --force parameter is passed.
I can see recordv3.py that has RecordAddCommand.

Comment (kihneman) [reply keepercommander/commands/record.py]: There is a check for a recordv3 title here: https://github.com/Keeper-Security/Commander/blob/ff396298ad720006b4ba7a33b9b86549012552af/keepercommander/commands/recordv3.py#L375

File: keepercommander/commands/record.py [hunk 2]
```diff
@@ -215,6 +215,9 @@ def execute(self, params, **kwargs):
                 password = input('...' + 'Password: '.rjust(16))
             if not url:
                 url = input('...' + 'Login URL: '.rjust(16))
+        if not title:
+            if len(title) == 0:
+                raise CommandError('add', 'Invalid title. Expected non-empty string.')
```
Comment (sk-keeper) [inline keepercommander/commands/record.py]: My Vault> add -v2 -f                                                                                                                                                                                                                                  
An unexpected error occurred: object of type 'NoneType' has no len(). Toggle debug to print traceback

Comment (kihneman) [reply keepercommander/commands/record.py]: I didn't mean to have the second conditional in there.

**Review by sk-keeper — APPROVED** [review]

**Pull request merged**